dd-sdk-android-fresco/src/main/kotlin/com/datadog/android/fresco/DatadogFrescoCacheListener.kt

```diff
@@ -0,0 +1,91 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.fresco
+
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.RumErrorSource
+import com.facebook.cache.common.CacheEvent
+import com.facebook.cache.common.CacheEventListener
+import com.facebook.cache.common.CacheKey
+
+/**
+ * Provides an implementation of [CacheEventListener] already setting up relevant Datadog triggers
+ * and information.
+ *
+ * It will automatically send RUM error events whenever writing to or reading from Cache throws
+ * a [java.io.IOException].
```

Comment:
```suggestion
 * Provides an implementation of [CacheEventListener] already set up to send relevant information 
 * to Datadog.
 *
 * It will automatically send RUM Error events whenever a read or write cache operation throws an exception.
```